obs_encoder.py

```diff
@@ -159,7 +159,7 @@ def __init__(self, obs_space:ObservationSpace, output_size=256, hidden_size=256)
             'supporter': self.supporter_encoder
         }
         # merge
-        obs_input_sizes = {
+        self.obs_merge_input_sizes = {
```

Comment:
Because I want to emphasize that this is the size of each encoded_obs before entering the merge fusion, I think it will be used in the policy_head, which is easy to distinguish from the obs_embedding after fusion